hal_st/stm32fxxx/UartStmDma.cpp

```diff
@@ -6,61 +6,61 @@ namespace hal
 {
     namespace
     {
-        volatile void* TransmitRegister(uint8_t uartIndex)
+        volatile void* TransmitRegister(infra::MemoryRange<USART_TypeDef* const> table, uint8_t uartIndex)
```

Comment:
I think we don't need this function anymore. The code below uses uartArray[uartIndex]->CR3, for example. Using uartArray[uartIndex]->TDR is more clear than TransmitRegister(uartArray, uartIndex)

Reply:
I have removed the function and replaced it with a member variable defined as a macro.